base/applications/notepad/lang/pt-PT.rc

```diff
@@ -22,7 +22,7 @@ END
 
 MAIN_MENU MENU
 BEGIN
-    POPUP "&Arquivo"
+    POPUP "&Ficheiro"
```

Comment:
for the record: this fixes the collision between "&Arquivo" and POPUP "&Ajuda".

Reply:
Can you share a pic of the pt-PT 2k3sp2 regedit where the menubar is visible?

Reply:
YES. the accelerator used for POPUP "&Ajuda" the same in 2k3sp2?